src/kfactory/packing.py

```diff
@@ -0,0 +1,79 @@
+"""Utilities for packing KCells and Instances."""
+
+from collections.abc import Sequence
+
+import rpack  # type: ignore[import-untyped,unused-ignore]
```

Comment:
how about moving this import inside the function, so that rpack is an optional dependency?

Reply:
And what would we gain of it not being a dependency?